moped-editor/src/views/projects/projectsListView/useProjectListViewQuery/useAdvancedSearch.js

```diff
@@ -73,7 +73,17 @@ const makeAdvancedSearchWhereFilters = (filters) =>
 
           // If it is a number or boolean, it does not need quotation marks
           // Otherwise, add quotation marks for the query to identify as string
-          value = type in ["number", "boolean"] ? value : `"${value}"`;
+          if (["number", "boolean"].includes(type)) {
+            // eslint-disable-next-line no-self-assign
+            value = value;
+          } else if (type === "array") {
+            // a virtue of this stringification of an array, we actually support
+            // comma delimited lists of districts and `graphql-engine` will do the
+            // correct set algebra
+            value = `[${value}]`;
+          } else {
+            value = `"${value}"`;
+          }
```

Comment:
Please see main PR comment, but short-story here is that `in` isn't doing what we thought it was. I believe `.includes()` fills the role of checking if an element exists in a list.